public/scripts/project.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+function Project(name, description, url, backgroundimage) {
+  this.name = name;
+  this.description = description;
+  this.url = url;
+  this.backgroundimage = backgroundimage;
+}
+
+Project.all = [];
+
+Project.prototype.toHtml = function() {
+  let template = Handlebars.compile($('#project-template').text());
+  return template(this);
+};
+
+Project.load = function(projectData){
+  projectData.forEach(function(proj) {
+    console.log(proj);
+    Project.all.push(new Project(proj));
+  })
+}
+
+Project.fetch = function() {
+  if(localStorage.projectData){
+    Project.load(JSON.parse(localStorage.projectData));
+    projectView.initIndexPage();
+  } else {
+    $.ajax({
+      url: './data/project_file.json',
+      dataType: 'json',
+      error: function (jqXHR, exception) {
```

Comment:
Great job with your error handling. If you wanted to consolidate some of this code, you can use switch statements.